File: validador.py
import re

def validar_email(email):
    padrao = r'^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
    return re.match(padrao, email)

def validar_cpf(cpf):
    padrao = r'^\d{3}\.\d{3}\.\d{3}-\d{2}$'
    return re.match(padrao, cpf)

def validar_telefone(telefone):
    padrao = r'^\(\d{2}\)\s\d{4,5}-\d{4}$'
    return re.match(padrao, telefone)


email_valido = validar_email('exemplo@email.com')
cpf_valido = validar_cpf('123.456.789-10')
telefone_valido = validar_telefone('(99) 99999-9999')

if email_valido:
    print('Email válido!')
else:
    print('Email inválido!')

if cpf_valido:
    print('CPF válido!')
else:
    print('CPF inválido!')

if telefone_valido:
    print('Número de telefone válido!')
else:
    print('Número de telefone inválido!')

File: validador.js
function validarEmail(email) {
    // Expressão regular para validar email
    const padrao2 = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
    return padrao.test(email);
  }
  
  function validarCPF(cpf) {
    // Expressão regular para validar CPF
    const padrao = /^\d{3}\.\d{3}\.\d{3}-\d{2}$/;
    return padrao.test(cpf);
  }
  
  function validarTelefone(telefone) {
    // Expressão regular para validar número de telefone
    const padrao = /^\(\d{2}\)\s\d{4,5}-\d{4}$/;
    return padrao.test(telefone);
  }
  
  // Testando os validadores
  const emailValido = validarEmail('exemplo@email.com');
  const cpfValido = validarCPF('123.456.789-10');
  const telefoneValido = validarTelefone('(99) 99999-9999');
  
  if (emailValido) {
    console.log('Email válido!');
  } else {
    console.log('Email inválido1')
  }
  
  //aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
  


